azure-pipelines.yml

```diff
@@ -0,0 +1,33 @@
+# .NET Desktop
+# Build and run tests for .NET Desktop or Windows classic desktop solutions.
+# Add steps that publish symbols, save build artifacts, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/apps/windows/dot-net
+
+trigger:
+- master
+
+pool:
+  vmImage: 'windows-latest'
+
+variables:
+  solution: '**/*.sln'
+  buildPlatform: 'Any CPU'
+  buildConfiguration: 'Release'
+
+steps:
+- task: NuGetToolInstaller@1
```

Comment:
This actually bit me. This wont actually install the latest version of NuGet. I believe you will get version 4.3.0, not 5.x

Reply:
Used the example provided to specify NuGet 5.x.

Regarding our convo on the other PR, since it seems like MS is eventually going to deprecate Azure Releases for general Pipelines, would it make sense for me to add the NuGet packing and pushing here, or in its own commit after this is merged in?

Reply:
I am not sure what MS is deprecating.... There is work to re-arrange how pipelines are done, but nothing is really going away just getting cleaned up.
This build pipeline should produce a nuget package, and we should have a separate release pipeline that is responsible for pushing it out. You want a separation of concerns between produce and release.

Reply:
I'll rephrase; the banner here: https://docs.microsoft.com/en-us/azure/devops/pipelines/release/?view=azure-devops _implies_ that they might get rid of release pipelines in the future, but it's kind of ambiguous, since the "new" pipelines appear to support multiple stages.

So to rephrase: do you think it's worth using the "classic" Release Pipeline, or stick with using YML in "Azure Pipelines". If that's not confusing... :)

Reply:
I think what that is in reference to, is that they are wanting to move release pipelines to yaml based ones. So in one sense the current pipelines are going away, but the functionality of having a separate release pipeline is not likely to be removed at all.

Reply:
Okay, gotcha. I'll add the step to generate the NuGet package before merging in, but I won't deploy it in this pipeline.